docs/.wci.yml

```diff
@@ -0,0 +1,57 @@
+---
+# Metadata parsed by the workflow community initiative, https://workflows.community/systems
+# Registered at https://github.com/workflowscommunity/workflowscommunity.github.io/blob/main/_data/workflow_systems.yml
+name: nextflow
+
+headline: A DSL for data-driven computational pipelines
+
+description: |
+  Nextflow is a bioinformatics workflow manager that enables the development of portable and reproducible workflows. It supports deploying workflows on a variety of execution platforms including local, HPC schedulers, AWS Batch, Google Cloud Life Sciences, and Kubernetes. Additionally, it provides support for manage your workflow dependencies through built-in support for Conda, Docker, Podmand, Singularity, and Modules.
+
+language: Groovy
+
+documentation:
+  general: http://docs.nextflow.io
+  installation: https://www.nextflow.io/docs/latest/getstarted.html
+  tutorial: https://training.seqera.io
+
+social:
+  twitter: nextflowio
+  youtube: https://www.youtube.com/c/Nextflow
+
+execution_environment:
+  interfaces:
+    - GitHub
+    - GitLab
+    - BitBucket
+  resource_managers:
+    - Slurm
+    - LSF
+    - PBS/Torque
+    - PBS Pro
+    - SGE
+    - Moab
+    - HTCondor
+    - Bridge
+    - HyperQueue
+    - GLS
+    - NQSII
+    - OAR
+    - Local
+    - AWS
+    - GCP
+    - Azure
+    - Kubernetes
+    - Docker
+    - Podman
+    - Singularity
+    - CharlieCloud
```

Comment:
```suggestion
```

Those are not resource manager

Reply:
You're correct, but it's been used by other workflow managers as a place to throw technologies they support 👀 

See [CWL](https://workflows.community/systems/common-workflow-language/) with Singularity or [PyCOMPSs](https://workflows.community/systems/compss/) putting Docker as an interface 😅.

It's a bit messy, and because it's in the repository of the projects, the workflow.community website doesn't have much control... But yeah, it looks fishy sticking the container technologies in this field.